docs/source/whatsnew.rst

```diff
@@ -1,6 +1,37 @@
 What's new in Hyperledger Fabric v2.x
 =====================================
 
+What's New in Hyperledger Fabric v2.4
+-------------------------------------
+
+Fabric Gateway
+^^^^^^^^^^^^^^
+
+The Fabric Gateway is a new service running within peer nodes that enables simplified transaction submission with the following benefits:
```

Comment:
just "Fabric Gateway is a new ..." ... as a feature name we don't use "The"

Reply:
running "on" not within peer nodes right. 

"... running on peer nodes that simplifies transaction submission and processing for the client application, with the following benefits:"